scripts/install

```diff
@@ -58,10 +58,13 @@ ynh_app_setting_set $app with_enigma $with_enigma
 #=================================================
 
 # jessie-backports is needed for php-net-ldap3
-grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
-  echo "deb http://httpredir.debian.org/debian jessie-backports main" \
-    | tee -a /etc/apt/sources.list.d/backports.list >/dev/null
-}
+if test -f /etc/apt/sources.list -o -d /etc/apt/sources.list.d
```

Comment:
Both file and directory exist on a YunoHost server, so why bother with that test?

Reply:
I had a grep error when I tried to install Roundcube through this package on a Debian 8 "Jessie" x86-64 from Gandi IaaS. I did not understand why, but this does not hurt. Moreover, it could be useful if one day YunoHost supports a system that does not use APT.